v4/source/groups.yaml

```diff
@@ -813,3 +813,32 @@
             $ref: "#/components/responses/InternalServerError"
           "501":
             $ref: "#/components/responses/NotImplemented"
+  "/users/{user_id}/groups":
+    get:
+      tags:
+        - groups
+      summary: Get groups for a userId
+      description: |
+        Retrieve the list of groups associated to the user
+
```

Comment:
Can you please update the doc to specify the permissions required to access this API

```
##### Permissions
 Must have `manage_system` permission or can access only for current user
```